### PR TITLE
Add boilerplate verification

### PR DESCRIPTION
**Pull request by pmorie**
Add boilerplate verification and instructions on using this repository.

cc @sarahnovotny

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,52 @@ This repository contains repository infrastructure tools for use in
...
+After this command, you will have:
+
+1.  A `repo-infra` directory in your project containing the content of **this**
+    project
+2.  2 new commits in the active branch:
+  1.  A commit that squashes the git history of the `repo-infra` project
+  2.  A merge commit whose ancestors are:
+    1.  The `HEAD` of the branch prior to when you ran `git subtree add`
+    2.  The commit containing the squashed `repo-infra` commits
+
```
Comment (eparis) [inline README.md]: How do I update?

File: verify/boilerplate/boilerplate.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def file_passes(filename, refs, regexs):
         (data, found) = p.subn("", data, 1)
 
     # remove shebang from the top of shell files
-    if extension == "sh":
+    if extension == "sh" or extension == "py":
```
Comment (eparis) [inline verify/boilerplate/boilerplate.py]: Aren't we careful about python shebanging elsewhere?

Comment (eparis) [inline verify/boilerplate/boilerplate.py]: This was added for a reason. Why are you removing it? Because you odn't want to fix your python?

Comment (pmorie) [inline verify/boilerplate/boilerplate.py]: I have a script that uses unbuffered (-u), which there doesn't seem to be a good way to programmatically use without setting the command line arg.

Comment (eparis) [inline verify/boilerplate/boilerplate.py]: The enforcement of `/usr/bin/env` needs to be done. Maybe it doesn't belong in the license check, but it should be done...

Comment (pmorie) [inline verify/boilerplate/boilerplate.py]: I'm cool with that, but I want to get this in for now.  Follow up?

File: README.md (hunk 1 above)
Comment (pmorie) [inline README.md]: I'm going to write that part in a follow-up

**Pull request merged**